File: apps/my-service-admin/src/api/order/Order.ts
import { JsonValue } from "type-fest";
import { Customer } from "../customer/Customer";

export type Order = {
  id: string;
  createdAt: Date;
  updatedAt: Date;
  orderDate: Date | null;
  status?: "Option1" | null;
  products: JsonValue;
  customer?: Customer | null;
};

File: apps/furniture-store/src/APIs/Order/Dtos/Order.cs
namespace FurnitureStore.APIs.Dtos;

public class Order
{
    public string Id { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }

    public double? TotalAmount { get; set; }

    public DateTime? OrderDate { get; set; }

    public string? Customer { get; set; }
}
